contributors/devel/adding-an-APIGroup.md

```diff
@@ -21,8 +21,11 @@ in your group;
 
 2. Create pkg/apis/`<group>`/{register.go, `<version>`/register.go} to register
 this group's API objects to the encoding/decoding scheme (e.g.,
-[pkg/apis/authentication/register.go](../../pkg/apis/authentication/register.go) and
-[pkg/apis/authentication/v1beta1/register.go](../../pkg/apis/authentication/v1beta1/register.go);
+[pkg/apis/authentication/register.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/authentication/register.go) and
+[pkg/apis/authentication/v1beta1/register.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/authentication/v1beta1/register.go);
+The register files must have a var called SchemeBuilder for the
+generated code to reference. There must be an AddToScheme method for
+the installer to reference.
```

Comment:
Optional: suggest you add something like, "You can look at a group under `pkg/apis/...` for example register.go files to use as a template, but do not copy the register.go files under `pkg/api/...`--they are not general."